### PR TITLE
[Main] Fix links for when deployed to microsoft/github.io/ApplicationInsights-JS

### PR DESCRIPTION
**Pull request by MSNev**

### CODE REVIEW

File: docs/Dependency.md [hunk 1]
```diff
@@ -4,15 +4,15 @@ Dependencies Plugin for the Application Insights Javascript SDK
 
 ## Configuration
 
-[`ICorrelationConfig`](../../shared/AppInsightsCommon/src/Interfaces/ICorrelationConfig.ts)
+[`ICorrelationConfig`](https://github.com/Microsoft/ApplicationInsights-JS/shared/AppInsightsCommon/src/Interfaces/ICorrelationConfig.ts)
```
Comment (MSNev) [inline docs/Dependency.md:7]: needs to include the blob/<branch>

File: docs/Dependency.md [hunk 2]
```diff
@@ -4,15 +4,15 @@ Dependencies Plugin for the Application Insights Javascript SDK
...
 
 ## Functions
 
 ### Dependency Listeners
 
 `addDependencyListener(dependencyListener: DependencyListenerFunction): IDependencyListenerHandler`
 
-A [dependency listener](../API-reference.md#adddependencylistener) is a callback function that **allows you to perform additional manipulation of the request details before the request is performed**.
+A [dependency listener](https://github.com/Microsoft/ApplicationInsights-JS/API-reference.md#adddependencylistener) is a callback function that **allows you to perform additional manipulation of the request details before the request is performed**.
```
Comment (MSNev) [inline docs/Dependency.md:15]: same

File: docs/Dependency.md [hunk 3]
```diff
@@ -54,7 +54,7 @@ handler.remove();
 
 `addDependencyInitializer(dependencyInitializer: DependencyInitializerFunction): IDependencyInitializerHandler`
 
-A [Dependency Initializer](../API-reference.md#adddependencyinitializer)  is very similar to a [Telemetry Initializer](https://github.com/Microsoft/ApplicationInsights-JS#telemetry-initializers) in that it **allows you modify the contents of collected telemetry before being sent from the user's browser**. And you can also returning `false` to cause the event to not be emitted.
+A [Dependency Initializer](https://github.com/Microsoft/ApplicationInsights-JS/API-reference.md#adddependencyinitializer)  is very similar to a [Telemetry Initializer](https://github.com/Microsoft/ApplicationInsights-JS#telemetry-initializers) in that it **allows you modify the contents of collected telemetry before being sent from the user's browser**. And you can also returning `false` to cause the event to not be emitted.
```
Comment (MSNev) [inline docs/Dependency.md:57]: same

File: docs/Dependency.md [hunk 4]
```diff
@@ -82,12 +82,12 @@ handler.remove();
 
 `trackDependencyData(dependency:` `IDependencyTelemetry``, properties?: { [key: string]: any }): void`
 
-[TrackDependencyData](../API-reference.md#trackdependencydata) function allows you to manually log a dependency call.
+[TrackDependencyData](https://github.com/Microsoft/ApplicationInsights-JS/API-reference.md#trackdependencydata) function allows you to manually log a dependency call.
```
Comment (MSNev) [inline docs/Dependency.md:85]: same

File: docs/Dependency.md [hunk 5]
```diff
@@ -82,12 +82,12 @@ handler.remove();
...
 
 ```javascript
 appInsights.trackDependencyData({absoluteUrl: 'some url', responseCode: 200, method: 'GET', id: 'some id'});
 ```
 
 ## Sample
 
-A [sample](./../examples/dependency/README.md) is provided to show how to filter, modify, block and disable dependency data.
+A [sample](https://github.com/Microsoft/ApplicationInsights-JS/examples/dependency/README.md) is provided to show how to filter, modify, block and disable dependency data.
```
Comment (MSNev) [inline docs/Dependency.md:93]: same

**Review by Karlie-777 — APPROVED** [review]

**Pull request merged**